src/components/PromotionList.tsx

```diff
@@ -26,26 +21,6 @@ import { Promotion, Restaurant, VoteState } from '../types/promotion';
 
 const PAGE_SIZE = 10;
 
-const styles: { [identifier: string]: CSSProperties } = {
-  container: {
-    backgroundColor: '#FFEDDC',
-    padding: 15,
-    overflow: 'auto',
-    scrollBehavior: 'smooth',
-    textAlign: 'center',
-  },
-  spinner: {
-    alignItems: 'center',
-    display: 'flex',
-    justifyContent: 'center',
-    height: '100%',
-    width: '100%',
-  },
-  spinnerIcon: {
-    fontSize: '4em',
-  },
-};
-
 /**
  * Selects the promotions that should be displayed on the current page.
  *
```

Comment:
I just realized we don't show the discount for a promotion, did something happen?
![image](https://user-images.githubusercontent.com/49849754/114284447-dd671880-9a04-11eb-8b2e-46b9e3856e7f.png)

Reply:
Oh I don't remember seeing the discount for a promotion! Here's what the designs look like:

<img width="298" alt="Screen Shot 2021-04-10 at 2 19 59 PM" src="https://user-images.githubusercontent.com/44531733/114284897-f32a0d00-9a07-11eb-901e-95c4a3f9e0c2.png">

Reply:
Hmm, it makes sense to show it right? How much effort would it take?

Reply:
@ericrkuo It shouldn't be too difficult, I think we might have to hold off on it for now because a concern from design was that the photo was too tall and that was because the length of the details in the promotion. I agree though it's a good idea to show so I've brought it up to them and they're still thinking about it!

Reply:
Ohh ok gotcah makes sense